src/EditorFeatures/CSharpTest/EditAndContinue/TopLevelEditingTests.cs

```diff
@@ -211,172 +218,6 @@ namespace N
-    }
-
-    public class D
-    {
-        public void M()
-        {
-            var d = new DirectoryInfo(""aa"");
-            var x = directoryInfo.Attributes;
-        }
-    }
-}";
-            var src2 = @"
-using System.IO;
-
-namespace N
-{
-    public class C
-    {
-        public C(string a) { }
-        public FileAttributes Attributes { get; set; }
-    }
-
-    public class D
-    {
-        public void M()
-        {
-            var d = new DirectoryInfo(""aa"");
-            var x = directoryInfo.Attributes;
-        }
-    }
-}";
-
-            var edits = GetTopEdits(src1, src2);
-
-            edits.VerifyEdits("Delete [using DirectoryInfo = N.C;]@20");
-
-            edits.VerifySemantics();
-        }
-
-        [Fact]
-        public void Using_Insert_ForNewCode()
-        {
-            // As distinct from the above, this test validates a real world scenario of inserting a using directive
-            // and changing code that utilizes the new directive to some effect.
-            var src1 = @"
-namespace N
-{
-    class Program
-    {
-        static void Main(string[] args)
-        {
-        }
-    }
-}";
-            var src2 = @"
-using System;
-
-namespace N
-{
-    class Program
-    {
-        static void Main(string[] args)
-        {
-            Console.WriteLine(""Hello World!"");
-        }
-    }
-}";
-
-            var edits = GetTopEdits(src1, src2);
-
-            edits.VerifySemantics(SemanticEdit(SemanticEditKind.Update, c => c.GetMember("N.Program.Main")));
-        }
-
-        [Fact]
-        public void Using_Delete_ForOldCode()
-        {
-            var src1 = @"
-using System;
-
-namespace N
-{
-    class Program
-    {
-        static void Main(string[] args)
-        {
-            Console.WriteLine(""Hello World!"");
-        }
-    }
-}";
-            var src2 = @"
-namespace N
-{
-    class Program
-    {
-        static void Main(string[] args)
-        {
-        }
-    }
-}";
-
-            var edits = GetTopEdits(src1, src2);
-
-            edits.VerifySemantics(SemanticEdit(SemanticEditKind.Update, c => c.GetMember("N.Program.Main")));
-        }
-
-        [Fact]
-        public void Using_Insert_CreatesAmbiguousCode()
-        {
-            // This test validates that we still issue edits for changed valid code, even when unchanged
-            // code has ambiguities after adding a using.
-            var src1 = @"
-using System.Threading;
-
-namespace N
-{
-    class C
-    {
-        void M()
-        {
-            // Timer exists in System.Threading and System.Timers
-            var t = new Timer(s => System.Console.WriteLine(s));
-        }
-    }
-}";
-            var src2 = @"
-using System.Threading;
-using System.Timers;
-
-namespace N
-{
-    class C
-    {
-        void M()
-        {
-            // Timer exists in System.Threading and System.Timers
-            var t = new Timer(s => System.Console.WriteLine(s));
```

Comment:
>record C [](start = 25, length = 8)

nit: formatting

Reply:
The merge has meant these comments point to the wrong spot, but I think I got them all.